modules/wildfly/pom.xml

```diff
@@ -51,6 +51,13 @@
             <groupId>org.hibernate.ogm</groupId>
             <artifactId>hibernate-ogm-infinispan</artifactId>
             <optional>true</optional>
+            <exclusions>
+                <exclusion>
+                    <!-- The Lucene dependency is not compatible with Neo4j embedded -->
+                    <groupId>org.hibernate</groupId>
+                    <artifactId>hibernate-search-engine</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
In this pull request, the versions of Lucene are not aligned. Neo4j is using 5.5.0 while HIbernate Search is using 5.3.1